app/docs/src/reference/function.md

```diff
@@ -1287,12 +975,12 @@ async function main({ thisComponent, components }: Input): Promise<Output> {
 }
 ```
 
-### Configuring pre-existing components
+### Configuring Existing Components
 
-An asset with a management component attached to it can have
-`management edges` to other types of components that it is allowed to manage.
-These management edges are the component context the function can act upon
-and allow those components to be configured.
+An asset with a management component attached to it can have `management edges`
+to other types of components that it is allowed to manage. These management
+edges are the component context the function can act upon and allow those
+components to be configured.
```

Comment:
We want to say "managed by" or "manages" since the word "edge" is now out of the product since sockets and the old UI died.